src/Shell/Task/MigrationSnapshotTask.php

```diff
@@ -79,7 +79,7 @@ public function templateData()
             'collection' => $collection,
             'tables' => $tables,
             'action' => 'create_table',
-            'name' => $this->BakeTemplate->viewVars['name'],
+            'name' => $this->Template->viewVars['name'],
```

Comment:
@dakota I suspect your version of bake is out of date.

[BakeTemplate](https://github.com/cakephp/bake/blob/master/src/Shell/Task/SimpleBakeTask.php#L31) is the task for rendering bake templates, [Template](https://github.com/cakephp/bake/blob/master/src/Shell/Task/TemplateTask.php) is the task for creating application/plugin template files (it used to be called View).